op-devstack/sysgo/l2_el_opgeth.go

```diff
@@ -108,13 +114,23 @@ func (n *OpGeth) Start() {
 	}
 
 	require := n.p.Require()
+
+	host, _, err := net.SplitHostPort(n.listenAddr)
+	require.NoError(err)
+
+	nt, err := nat.Parse(fmt.Sprintf("extip:%s", host))
```

Comment:
I think extip should be optional, so we may use `n.listenAddr` in default `nil`, and use extip when it is not nil. Also this should be implemented also for `l2_el_opreth.go`.